### PR TITLE
[CHEF-6088 CHEF-6094 CHEF-6107] Cookie policy, Privacy policy, logo and favicon changes

### PR DESCRIPTION
**Pull request by ayush-billore**
### Description

- Adds cookie policy
- Update logo
- Update  favicon

https://chefio.atlassian.net/browse/CHEF-6088
https://chefio.atlassian.net/browse/CHEF-6094
https://chefio.atlassian.net/browse/CHEF-6107

Screenshot 2023-11-22 at 3 43 57 PM

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussions that are relevant]

### Check List

- [ ] New functionality includes tests
- [ ] All buildkite tests pass
- [ ] Full omnibus build and tests in buildkite pass
- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/main/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

**Comment by RoyShravani** [comment]
LGTM!

**Review by RoyShravani — APPROVED** [review]

File: src/oc-id/lib/assets/images/progress-chef-logo.png
Comment (RajeshPaul38) [inline src/oc-id/lib/assets/images/progress-chef-logo.png:1]: This file is not being used. We can skip it in that case.

Comment (ayush-billore) [reply src/oc-id/lib/assets/images/progress-chef-logo.png:1]: @RajeshPaul38  I added both png and svg as earlier also there were both versions.

**Review by saghoshprogress — APPROVED** [review]

**Review by marcparadise — APPROVED** [review]

**Comment by marcparadise** [comment]
This looks good, but probably needds a rebase on main to get  verify tests passing.

**Review by marcparadise — CHANGES_REQUESTED** [review]
This looks fine, but needs a rebase to getverification passing. 

Please also update the commit message to include a summary of the change and the reason the change is being implemented. ..

**Comment by ayush-billore** [comment]
> This looks fine, but needs a rebase to getverification passing.
> 
> Please also update the commit message to include a summary of the change and the reason the change is being implemented. ..

Thanks for the suggestion @marcparadise 

I have added following comment -

```
The PR adds following GDPR complaiance related changes to OCID component
- add cookie policy to the footer
- update logos to reflect current Progress Chef brand.
- update favicon to reflect current Progress Chef brand.

Signed-off-by: ayush-billore <ayush.billore@progress.com>
```

**Review by jashaik — APPROVED** [review]

**Pull request merged**